packages/cli/commands/project/dev.js

```diff
@@ -96,19 +100,27 @@ exports.handler = async options => {
   let targetAccountId = options.account ? accountId : null;
   let createNewSandbox = false;
   const defaultAccountIsSandbox = isSandbox(accountConfig);
+  const defaultAccountIsDeveloperTestAccount = isDeveloperTestAccount(
+    accountConfig
+  );
 
-  if (!targetAccountId && defaultAccountIsSandbox) {
+  if (
+    !targetAccountId &&
+    (defaultAccountIsSandbox || defaultAccountIsDeveloperTestAccount)
+  ) {
     logger.log();
-    const useDefaultSandboxAccount = await confirmDefaultSandboxAccountPrompt(
+    const useDefaultAccount = await confirmDefaultAccountPrompt(
```

Comment:
I renamed this to be more generic since the copy in the confirmation could be used for both sandbox and dev test account